CMakeLists.txt

```diff
@@ -16,9 +16,8 @@ FetchContent_Declare(
 )
 FetchContent_MakeAvailable(Catch2)
 
-add_library(copper INTERFACE include/copper.h)
```

Comment:
Thanks for contributing. 
What's the reasoning for this change? I realize it doesn't actually do anything for compilation but IDEs sometimes assume that you actually declared all source and header files.

Reply:
I am getting a failure to build from the command line. The IDEs should infer the list of used headers from the compilation database.

https://cmake.org/cmake/help/latest/command/add_library.html#interface-libraries shows that defining an **interface** library does not accept source elements.

Reply:
According to that link, that functionality was added in 3.19. But considering you downgraded the version requirement to 3.18, I assume that's not an option for you? In that case, you could use [target_sources](https://cmake.org/cmake/help/latest/command/target_sources.html#command:target_sources) instead, which should work in older CMake.

Reply:
That's interesting, I wasn't aware of it.

VS 2019 comes with CMake 3.19 so I'll try upgrading CMake on this build server.